solidity/contracts/KeepRandomBeaconOperator.sol

```diff
@@ -476,15 +477,23 @@ contract KeepRandomBeaconOperator is ReentrancyGuard {
         // slashing in case of any changes in .call() gas limit.
```

Comment:
We probably don't need `uint256 callbackFee = currentRequestCallbackFee;` in line 468 and we can use `currentRequestCallbackFee` directly. It should let us save some gas/space.

Reply:
Removed in https://github.com/keep-network/keep-core/pull/1745/commits/5de53849ac781bdef8aae22b7d5da0bc01352604